storybookjs/storybook
# Addon-docs/Angular: Render user defined template as source if it exists

Issue: https://github.com/storybookjs/storybook/issues/14268

## What I did

Added a  flag to indicate that the story has a custom template, and render that instead of a generated template

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

Added a new story for the doc-button component, which uses a template. Also several other stories have custom templates, and can be used for verification
